README.md

```diff
@@ -1,11 +1,21 @@
 # DCISolver - Dynamic Control of Infeasibility Solver
 
-![CI](https://github.com/JuliaSmoothOptimizers/DCISolver.jl/workflows/CI/badge.svg?branch=master)
-[![codecov](https://codecov.io/gh/JuliaSmoothOptimizers/DCISolver.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/JuliaSmoothOptimizers/DCISolver.jl)
 [![GitHub](https://img.shields.io/github/v/release/JuliaSmoothOptimizers/DCISolver.jl.svg?style=flat-square)](https://github.com/JuliaSmoothOptimizers/DCISolver.jl/releases)
```

Comment:
Hey @amontoison ! Thanks for the PR. It is a great idea to uniformize this. Wouldn't it be better with the release also included in the table?

Reply:
I added it in the table.